internal/internal_workflow_client.go

```diff
@@ -1669,55 +1734,105 @@ func (w *workflowClientInterceptor) ExecuteWorkflow(
 	}, nil
 }
 
-func (w *workflowClientInterceptor) executeWorkflowWithOperation(
+func (w *workflowClientInterceptor) UpdateWithStartWorkflow(
 	ctx context.Context,
-	startRequest *workflowservice.StartWorkflowExecutionRequest,
-	operation WithStartWorkflowOperation,
-) (*workflowservice.StartWorkflowExecutionResponse, error) {
-	startOp := &workflowservice.ExecuteMultiOperationRequest_Operation{
-		Operation: &workflowservice.ExecuteMultiOperationRequest_Operation_StartWorkflow{
-			StartWorkflow: startRequest,
-		},
+	updateInput *ClientUpdateWorkflowInput,
+	startOperation WithStartWorkflowOperation,
+) (WorkflowUpdateHandle, error) {
+	startOp, ok := startOperation.(*WithStartWorkflowOperationImpl)
+	if !ok {
+		panic("startOperation must be created by NewWithStartWorkflowOperation")
```

Comment:
Might as well just return an error here same as you had pre-interception

Reply:
Thanks I had intended to convert all those panics to errors, but I missed this one.